### DISCUSSION TITLE
Jagrosh Music Bot Keeps Disconnecting Right After Connecting To Channel

### DISCUSSION THREAD

**Discussion by btrade890** [jagrosh/MusicBot#1620] [Troubleshooting]
I've been using the **Jagrosh Music Bot** for more than 4 months now and I've never had a problem with it.

but 3 days ago I faced this problem when I requested to play a song, the bot joined for a second and disconnected right away. I'm hosting the bot on **"Pebblehost"** servers so that I don't host it from my computer.

**Below is a screenshot of the log when I tried to play a song, I was in a channel and typed the command "/play faouzia ice".
Normally it works with no problem. but as I explained above in the past 3 days it has been connecting and disconnecting a second after connecting.**

Any suggestions on what I can do?

![image](https://github.com/user-attachments/assets/1a1a3208-d3c2-4972-8f19-fb9bab08c463)

**Comment by MichailiK** [comment]
PebbleHost’s IPs have been banned from playing back videos from YouTube without login. This issue is already tracked in #1588
